lib/logos.js

```diff
@@ -31,7 +32,12 @@ function prependPrefix(s, prefix) {
 }
 
 function isDataUrl(s) {
-  return s !== undefined && /^(data:)([^;]+);([^,]+),(.+)$/.test(s)
+  try {
+    Joi.assert(s, Joi.string().dataUri())
```

Comment:
Turns out we don't need wheel 2.0 here :) https://github.com/hapijs/joi/blob/master/API.md#stringdataurioptions